src/components/MainContent/MainContent.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useContext } from 'react';
+import {
+  ActivePostContext,
+  ActiveUserContext,
+  ErrorsContext,
+  LoaderContext,
+  PostsContext,
+} from '../../utils/Store';
+import { Loader } from '../Loader';
+import { PostsList } from './PostList/PostsList';
+import { ErrorTypes } from '../../types/ErrorTypes';
+
+export const MainContent: React.FC = () => {
+  const { activeUser } = useContext(ActiveUserContext);
+  const { activePost } = useContext(ActivePostContext);
+  const { posts } = useContext(PostsContext);
+  const { isError } = useContext(ErrorsContext);
```

Comment:
The variable `isError` should be checked against `ErrorTypes` to ensure it matches the expected error type. Make sure `ErrorTypes` is correctly imported and used.